recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -85,6 +86,9 @@ if not "%CUDA_PATH%" == "" (
     echo set "CUDA_PATH=%CUDA_PATH%"              >> "%CONDA_PREFIX%\etc\conda\activate.d\conda-forge-ci-setup-activate.bat"
     echo set "CUDA_HOME=%CUDA_PATH%"              >> "%CONDA_PREFIX%\etc\conda\activate.d\conda-forge-ci-setup-activate.bat"
 )
+if not "%CUDA_VERSION%" == "None" (
+    echo set "CONDA_OVERRIDE_CUDA=%CONDA_OVERRIDE_CUDA%" >> "%CONDA_PREFIX%\etc\conda\activate.d\conda-forge-ci-setup-activate.bat"
```

Comment:
You can move this line into the if-block above so it benefits from the CUDA_PATH check (which is testing essentially for the same: did we successfully install CUDA, which implies that CUDA_VERSION is not None).

Reply:
I think we have to keep the parts that write to `"%CONDA_PREFIX%\etc\conda\activate.d\conda-forge-ci-setup-activate.bat"` all together, so I don't think I can move this up any higher and eliminate the second "if CUDA" block. The first line in this section creates file with `>` and the following lines append with `>>`, so I can't know whether to create/append if I put the CUDA piece first (at the end of the first "if CUDA" block). Also we should verify the CUDA status before we start writing to that file. Does that make sense / do you have another solution in mind?

Reply:
I meant the block immediately above (lines 86-87).

Reply:
My mistake, I misread where the comment was pointing. Thanks, that makes complete sense now. 😆